meetings/2024-05-15.md

```diff
@@ -0,0 +1,134 @@
+* TBL: Websockets example: any breaking change has a negative impact on the ecosystem. They should be implemented rarely, and when they are ??? need a way to make sure we are backwards-compatible. Working with SAI is very different. 
+* eP: See with implementers and deployments - communication loop to be more aware of actual adoption. Harmful for the ecosystem but an argument is the ecosystem is slow so if need to make breaking changes now is the best time. Good to have strategy to be able to make breaking changes later, now is a good time to test drive deprecating, sunsetting, parts of the protocol. 
+* MdJ: We already have a mechanism. Introduce the new one, say the insecure one is deprecated.. i don't think that's really your question, Pavlik? 
+* TBL: If there's a breaking change number before dot should change. So we should use semver >= 1.0. 
+* MdJ: I think eP's question is not how do we number, how do we sunset.. it's more we have all these different implementations side by side which are not compatible with each other. How do we keep having a single spec even with contradictions? Some people work on WAC, some on ACP, some on SAI, some on Inrupt's access requests, etc. We have different ways of doing policy registries, how can we move to a spec that includes SAI? 
+* eP: For example, what if slash semantics go away? what's the strategy? Looking at things we'd like to anticipate changes on, how to communicate. 
+* MdJ: With Websockets we all agreed ??? We cannot say to make a breaking change without excluding ??? 
+* eP: I think it's a good exercise to say if a breaking change needs to happen, how to approach. 
+* MdJ: The WG will be very different from CG. WG will need to arrive at one protocol and CG is an incubator. If someone in the CG still wants to do X (e.g. use slash semantics) I don't think we should deprecate it. 
+* TBL: If people want to deprecate slash semantics, there would
+*  be a fork. If you wanted to make a spec that included ??? you'd have different levels of ??? that you can negociate between client and server. 
+* eP: We should clarify what's the difference between fork and next version with breaking changes. All specs in CG are CG drafts. More important for WG. WG publishes the actual recommendations. We need to clarify fork, breaking changes, CG drafts, recommendations from WG, etc. Those can have breaking changes, assuming everything would be a fork? We need a clear startegy. 
+* RG: TBL you're breaking up.
+* MdJ: TBLs and my answer maybe you have clarification/answer to your question if you add them together.
+* eP: We can move on. 
+* MdJ: To summarize, if the CG were to pick one of five parallel experiments as the next version of Solid then it'd be up to the experiment to say they're excluded. Because in 2 years there will be a version published by WG anyway, there's no use trying to converge before that happens. Let's stick with multiple options.
```

Comment:
```suggestion
* MdJ: To summarize, if the CG were to pick one of five parallel experiments as the next version of Solid, then it'd be up to the experiment to say they're excluded. Because in 2 years, there will be a version published by WG anyway, there's no use trying to converge before that happens. Let's stick with multiple options.
```